### DISCUSSION TITLE
Loss function for the LogisticRegressionGD (Ch. 3)

### DISCUSSION THREAD

**Discussion by eliloz** [rasbt/machine-learning-book#103] [Q&A]
In the code for calculating the loss for the LogisticRegressionGD, I noticed that the division by `X.shape[0]` is being applied only to the second term and not the whole equation:
```loss = -y.dot(np.log(output)) - ((1 - y).dot(np.log(1 - output))) / X.shape[0]```

Shouldn't this be `loss = ( -y.dot(np.log(output)) - ((1 - y).dot(np.log(1 - output))) ) / X.shape[0]` ?

Thanks!
